Mathlib/Algebra/Group/Defs.lean

```diff
@@ -274,87 +274,86 @@ instance Semigroup.to_isAssociative : IsAssociative G (· * ·) :=
 
 end Semigroup
 
-/-- A commutative semigroup is a type with an associative commutative `(*)`. -/
+/-- A commutative addition is a type with an addition which commutes-/
+@[ext]
+class AddComm (G : Type u) extends Add G where
+  /-- Addition is commutative in an additive commutative semigroup. -/
+  protected add_comm : ∀ a b : G, a + b = b + a
+
+/-- A commutative multiplication is a type with a multiplication which commutes-/
 @[ext]
-class CommSemigroup (G : Type u) extends Semigroup G where
+class CommMul (G : Type u) extends Mul G where
```

Comment:
These should be either `*Comm` or `Comm*`, not one of each!

Reply:
We have `CommMonoid`, `CommGroup` and `CommSemigroup` for multiplicative and `AddCommSemigroup`, `AddCommMonoid` and `AddCommGroup` for additive. So should this be `CommMul` and `AddCommMul` (which sounds wrong)? `to_additive` mapped `CommMul` to `AddComm` by default.

Reply:
@eric-wieser How about `CommMagma` and `AddCommMagma`? But the last time I used `Magma` I was told it should be `Mul`.

Reply:
Those names sound good to me

Reply:
Thanks, I've made that change.